tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -771,10 +772,10 @@ def fieldop_domain(a: Field[[IDim, JDim], float64]) -> Field[[IDim, JDim], float
     def program_domain(
         inp: Field[[IDim, JDim], float64],
         out: Field[[IDim, JDim], float64],
-        lower_i: int64,
-        upper_i: int64,
-        lower_j: int64,
-        upper_j: int64,
+        lower_i: int32,
```

Comment:
is this a strong requirement now? if yes, we should introduce an alias or maybe even a distinct type (at the frontend level).